.gitignore

```diff
@@ -2,3 +2,4 @@ _site/
 .sass-cache/
 .jekyll-cache/
 .jekyll-metadata
+/.idea/
```

Comment:
What is that change about?

Reply:
nothing changes. was using IDEA, and by default, it would generate the meta.